.github/workflows/ci.yml

```diff
@@ -14,6 +14,11 @@ jobs:
           - operating-system: 'ubuntu-latest'
             php-version: '8.1'
             composer-flags: '--ignore-platform-req=php'
+            phpunit-flags: '-d disable_functions=pcntl_signal_dispatch'
+          - operating-system: 'ubuntu-latest'
+            php-version: '8.1'
+            composer-flags: '--ignore-platform-req=php'
+            phpunit-flags: '-d disable_functions=pcntl_signal_dispatch'
```

Comment:
`pcntl_signal_dispatch` is now disabled in both runs.

Reply:
Also, could you please add the flags so they show up in the job name?

Reply:
Whoops, fixed :)